src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/MS/Internal/MarkupCompiler/VersionHelper.cs

```diff
@@ -0,0 +1,215 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Modified from https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/VersionHelper.cs
+
+#nullable enable
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace MS.Internal
+{
+    internal static class VersionHelper
+    {
+        static Version NullVersion = new Version(0, 0, 0, 0);
```

Comment:
And I think the NullVersion is a readonly field

Reply:
And the original code is `readonly` 

https://github.com/dotnet/roslyn/blob/284a67e9e9e1b33370aadf15ff082389f9dedb50/src/Compilers/Core/Portable/MetadataReference/AssemblyIdentity.cs#L248